src/widgets/auto-style/style-utils.js

```diff
@@ -1,21 +1,26 @@
 var _ = require('underscore');
 var postcss = require('postcss');
+var stripInlineComments = require('postcss-strip-inline-comments');
+var SCSSsyntax = require('postcss-scss');
```

Comment:
After a BIG research trying to find why this builds was failing I finally found that this is the guilty line!

For some reason when postcss-scss is included, jasmine is unable to find the tests breaking the build. (I thought it was something related to the CI speed improvement x.x )

I dont know the exact purpose of the PR but I see 3 options:

1. Try to use another library
2. Change the test runner
3. Try to find whats happening (trust me, this is the hardest!)